src/cargo/ops/cargo_new.rs

```diff
@@ -258,6 +258,13 @@ fn check_name(
             name
         ))?;
     }
+    let name_in_lowercase = name.to_lowercase();
+    if name != name_in_lowercase {
+        shell.warn(format!(
+            "package `{}` should have a lowercase name such as `{}`",
```

Comment:
In the last change, the wording became strong, going from talking of "preferred" to "should"

Reply:
Is this a better option:

"package {name} could have a lowercase name (preferred) such as {lower_name}"

Reply:
"could" feels bit weird

For brainstorming ideas, here are some options:
```rust
// soften with "generally" and focusing on the principle rather than this specific package
"package names should generally be snake_case or kebab-case, like `{name_in_lowercase}`, but got `{name}`
// Switch to "recommended" and offering an option
"`{name}` is not snake_case or kebab case which is recommended for package names, consider {name_in_lowercase}"
```

This is a lot harder when we don't have a specific lint to point to which provides just enough context for the original message to work.